doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -974,32 +974,42 @@ Generation of induction principles with ``Scheme``
    .. insertprodn scheme_kind sort_family
 
    .. prodn::
-      scheme_kind ::= Equality for @reference
-      | {| Induction | Minimality | Elimination | Case } for @reference Sort @sort_family
+      scheme_kind ::= {| Induction | Minimality | Elimination | Case } for @reference Sort @sort_family
       sort_family ::= Set
       | Prop
       | SProp
       | Type
 
-  A high-level tool for automatically generating
-  (possibly mutual) :term:`induction principles <induction principle>`
-  for given types and sorts.
-  Each :n:`@reference` is a different inductive type identifier belonging to
-  the same package of mutual inductive definitions.
-  The command generates the :n:`@ident`\s as mutually recursive
-  definitions. Each term :n:`@ident` proves a general principle of mutual
-  induction for objects in type :n:`@reference`.
+   Generates (possibly mutual) :term:`induction principles <induction principle>`
+   for given types and sorts. Each :n:`@reference` is a different inductive type
+   identifier belonging to the same package of mutual inductive definitions. The
+   command generates the :n:`@ident`\s as mutually recursive definitions. Each
+   term :n:`@ident` proves a general principle of mutual induction for objects
+   in type :n:`@reference`.
 
-  :n:`@ident`
-    The name of the scheme. If not provided, the scheme name will be determined automatically
-    from the sorts involved.
+   :n:`@ident`
+      The name of the scheme. If not provided, the scheme name will be determined
+      automatically from the sorts involved.
+
+   The following :n:`@scheme_kind`\s generate induction principles with
+   given properties:
+
+   =================== =========== ===========
+    :n:`@scheme_kind`   Dependent   Recursive
+   =================== =========== ===========
+    :n:`Induction`         Yes         Yes
+    :n:`Minimality`        No          Yes
+    :n:`Elimination`       Yes         No
+    :n:`Case`              No          No
+   =================== =========== ===========
+
+   See examples :ref:`here <scheme_example>`.
 
-  :n:`Minimality for @reference Sort @sort_family`
-    Defines a non-dependent elimination principle more natural for inductively defined relations.
+.. cmd:: Scheme Equality for @reference
```

Comment:
FWIW, it may be slightly confusing to have both "Scheme" and "Scheme Equality" appear in the index by default.  In particular, would you check the uses of ``` :cmd:`Scheme ``` and make sure they point to the right command?  I do see one usage with ``` :cmd:`Scheme` :n:`Equality …` ``` that should become ``` :cmd:`Scheme Equality` ```

Reply:
I believe that is the only usage.